src/test/java/com/ericsson/eiffel/remrem/publish/helper/RMQHelperUnitTest.java

```diff
@@ -92,6 +92,17 @@ private void cleanProperties() {
     	assertTrue(rmqHelper.getHost().equals(host));
     }
 
+    @Test public void testConnection() {
+        try {
+            rmqHelper.cleanUp();
+            if(!mockConnection.isOpen())
+                rmqHelper.createConnection();
```

Comment:
You are not testing anything here. Assert that you have the connection. You only test if connection is null, connection null is what you get from cleanup. Test also that you can create a connection after you do mockConnection.close()